service/configservice/api.go

```diff
@@ -4029,6 +4029,7 @@ func (s *DeliveryChannel) SetSnsTopicARN(v string) *DeliveryChannel {
 // The status of a specified delivery channel.
 //
 // Valid values: Success | Failure
+//
```

Comment:
For these i suggest just removing the api.go files from the PR. We can upstream these doc issues to the service teams